server/routerlicious/packages/lambdas/src/alfred/index.ts

```diff
@@ -619,6 +622,34 @@ export function configureWebSocketServices(
 				);
 			}
 
+			// Send signal to room from broadcast-signal endpoint
+			if (httpServer !== undefined && eventEmitter !== undefined) {
+				eventEmitter.on("broadcast-signal", (_, containerUrl) => {
+					const documentId = containerUrl.split("/")[containerUrl.split("/").length - 1];
+					const tenantId = containerUrl.split("/")[containerUrl.split("/").length - 2];
```

Comment:
I wonder if the right interface here is to have these pre-split for the lambda by the event emitter; this feels a bit too magical. Might accompany the use of TypedEventEmitter to have an actual interface for the event.

Reply:
+1 - good idea to get URLs out of the API surface as quickly as possible (and instead pass the strongly-typed parts that you care about).  This reduces the risk of weird alternate flows and better explains the api contrct